src/main/java/hudson/plugins/git/extensions/impl/CloneOption.java

```diff
@@ -49,13 +59,18 @@ public void decorateCloneCommand(GitSCM scm, Run<?, ?> build, GitClient git, Tas
             listener.getLogger().println("Using shallow clone");
             cmd.shallow();
         }
+        if (noTags) {
```

Comment:
Technically, I agree.
But I tried to follow the code style that was already there. In the whole class, private fields are accessed directly and not by using the public accessor methods. Just take a look at the preceding four lines or the next four lines to see what I mean.